### DISCUSSION TITLE
Does SkyWalking support multi-tenant mode？

### DISCUSSION THREAD

**Discussion by SimonLiuhd** [apache/skywalking#7926] [Q&A]
Dear all，

Does SkyWalking support multi-tenant mode？Or is there solution for the multi-tenant mode?

We have some systems deployed for different customers, and we want to deploy one SkyWalking system for the customers. But we require that different customer cannot touch the system information which is not belonged to them.

Any suggestion is appreciated.
Thanks in advance.

Simon

**Comment by wu-sheng** [comment]
Your customers, your responsibility. The community has no one working about this.
